lib/releng_common/setup.py

```diff
@@ -14,18 +14,25 @@
 with open(os.path.join(here, 'VERSION')) as f:
     version = f.read().strip()
 
-with open(os.path.join(here, 'requirements.txt')) as f:
-    install_requires = f.read().strip().split('\n')
-
-
 setup(
     name='releng_common',
     version=version,
     description='Services behind https://mozilla-releng.net',
     author='Mozilla Release Engineering',
     author_email='release@mozilla.com',
     url='https://github.com/garbas/mozilla-releng',
-    install_requires=install_requires,
+    install_requires=[
+        'Flask',
+        'Jinja2',
+    ],
+    extras_require=dict(
+        cache=['Flask-Cache'],
+        db=['Flask-SQLAlchemy', 'Flask-Migrate'],
+        auth=['Flask-Login', 'taskcluster'],
+        api=['connexion'],
+        log=['structlog'],
+        cors=['Flask-Cors'],
+    ),
```

Comment:
yes, not every service will use ``db`` or ``auth`` which would mean smaller docker images and faster deployment.